fdbserver/ptxn/test/TestTLogServer.actor.cpp

```diff
@@ -302,6 +308,80 @@ ACTOR Future<Void> startStorageServers(std::vector<Future<Void>>* actors,
 	return Void();
 }
 
+ACTOR Future<int> fetchTeamCount(std::shared_ptr<ptxn::TLogInterfaceBase> pInterface, ptxn::TLogGroupID tLogGroupID) {
+	ptxn::TLogTestInfoReply reply =
+	    wait(pInterface->testInfo.getReply(ptxn::TLogTestInfoRequest{ .fetch_group_info = true }));
+
+	auto it = reply.groupToTeams.find(tLogGroupID);
+	ASSERT(it != reply.groupToTeams.end());
+	int numTeams = it->second.size();
+	ASSERT(numTeams > 0);
+	ASSERT(reply.groupToTeams.count(tLogGroupID) > 0);
+	return reply.groupToTeams.find(tLogGroupID)->second.size();
```

Comment:
Seems redundant: the assert is the same as line 318, and the return value is just `numTeams`.